### PR TITLE
Flesh out x-audio readme

### PR DESCRIPTION
**Pull request by benbarnett**
Starter for ten for x-audio readme.

### CODE REVIEW

File: components/x-audio/readme.md [hunk 1]
```diff
@@ -1,6 +1,18 @@
 # x-audio
 
-This module has these features and scope.
+This module can be used to play an audio file via pluggable engines/players.
+
+It comes bundled with a web audio player (via [HTMLAudioElement](https://developer.mozilla.org/en-US/docs/Web/API/HTMLAudioElement)), but can be used with another engine such as in the native layer. The bundled engine uses a [Redux](https://redux.js.org/) to manage state.
+
+Redux was chosen as it is a well known method for updating state via distinct actions. x-audio implements middleware (found in `src/redux/player-logic.js`) which drives the [HTMLAudioElement](https://developer.mozilla.org/en-US/docs/Web/API/HTMLAudioElement). 
+
+For example, when redux receives a `PLAY` action, it will trigger `audioElement.play()`.
```
Comment (robsquires) [inline components/x-audio/readme.md]: is it `REQUEST_PLAY`

Comment (benbarnett) [reply components/x-audio/readme.md]: yep yep, will change

File: components/x-audio/readme.md [hunk 2]
```diff
@@ -23,16 +35,22 @@ The [`x-engine`][engine] module is used to inject your chosen runtime into the c
...
-// A == B == C
-const a = Audio(props);
-const b = <Audio {...props} />;
-const c = React.createElement(Audio, props);
+// Render the UI component as usual
+<Audio {...props} />;
+
+
+// OR, Import the component bundled with the html audio player with redux
+import ConnectedPlayer from '@financial-times/x-audio';
```
Comment (robsquires) [inline components/x-audio/readme.md]: the default export is function, which you run on your side to obtain the functional audio player component. Maybe we call it `createAudioPlayer`?
```
const AudioPlayer = createAudioPlayer();
<AudioPlayer {...props} />;
```

It's a function so you don't end up initialising the store just by importing it

Comment (robsquires) [reply components/x-audio/readme.md]: Maybe` initAudio` is a better name? 🤔

File: components/x-audio/readme.md [hunk 3]
```diff
@@ -46,8 +64,21 @@ All `x-` components are designed to be compatible with a variety of runtimes, no
...
+There are two types of property for x-audio. Ultimately these boil down to meta around the audio to be played, and `notifiers` to allow consumers to respond to state changes. They are detailed below.
+
+#### x-audio props
+
+Feature          | Type   | Notes
+-----------------|--------|----------------------------
+`title`  | String | The title of the audio, e.g. ‘May 23rd’
+`seriesName`  | String | The series name, e.g. ‘News in Brief’
+`url` | String | The url of the audio file, e.g. ‘https://acast.com/audio.mp3’
+`duration` | Number | The duration of the audio in seconds, e.g. ‘60’
```
Comment (robsquires) [inline components/x-audio/readme.md]: Could prob take that out given Asuka's duration PR?

File: components/x-audio/readme.md (hunk 2 above)
Comment (benbarnett) [reply components/x-audio/readme.md]: @robsquires that's what I thought but then got confused when I read this: https://github.com/Financial-Times/x-dash/blob/x-audio/components/x-audio/src/Audio.jsx#L6 🤔 

Hmm

File: components/x-audio/readme.md [hunk 4]
```diff
@@ -23,16 +35,22 @@ The [`x-engine`][engine] module is used to inject your chosen runtime into the c
 
 ## Usage
 
-The components provided by this module are all functions that expect a map of [properties](#properties). They can be used with vanilla JavaScript or JSX (If you are not familiar check out [WTF is JSX][jsx-wtf] first). For example if you were writing your application using React you could use the component like this:
+This component exports both the plain UI component and the default is the redux-connected component, ready to play audio.
```
Comment (robsquires) [inline components/x-audio/readme.md]: Yeah same, _default is a factory function which returns the working audio player (under the hood this is the plain UI component hooked up to the redux store)_

Comment (robsquires) [reply components/x-audio/readme.md]: something like that maybe ?^ 🤔

Comment (benbarnett) [reply components/x-audio/readme.md]: Yep, nice

**Pull request merged**